extensions/wikia/Insights/InsightsController.class.php

```diff
@@ -136,8 +150,8 @@ private function getCongratulationsNotificationParams() {
 	 * @param $next Array data about item from insight list
 	 * @param $params String params to be added to url
 	 */
-	private function getInsightFixedNotificationParams( $next, $subpage ) {
-		$params = $this->getInsightNextLinkParams( $next, $subpage );
+	private function getInsightFixedNotificationParams( $subpage, $articleName, $model ) {
```

Comment:
if params on the list are array or object of specific class please add this requirement on function params list.
(PHP type hinting http://php.net/manual/en/language.oop5.typehinting.php)